src/Web/Rollbar/Types.hs

```diff
@@ -2,21 +2,41 @@
 
 module Web.Rollbar.Types where
 
-import Control.Lens.TH
+import Control.Lens.TH (makeClassy, makeLenses)
 import Data.Aeson (Value)
+import Data.Text (Text)
+
+newtype APIToken = APIToken
```

Comment:
I'd like to see comments on the new types and fields in this file, hopefully with examples. For example, I currently have no idea what `CodeVersion` means, and I don't know what possible values are for `_eventContext`.

Reply:
Great suggestion! I added some Haddock documentation here: https://github.com/collegevine/rollbar/pull/3/commits/193a0c2c499f204489dfdfde96802a287c51bcad